core/controller/Dockerfile

```diff
@@ -1,13 +1,11 @@
 FROM scala
 
-ENV DEBIAN_FRONTEND noninteractive
-
 # Install swagger-ui
-RUN wget --no-verbose https://github.com/swagger-api/swagger-ui/archive/v2.1.4.tar.gz && \
+RUN curl -sSL -o swagger-ui-v2.1.4.tar.gz --no-verbose https://github.com/swagger-api/swagger-ui/archive/v2.1.4.tar.gz && \
     mkdir swagger-ui && \
-    tar zxf v2.1.4.tar.gz -C /swagger-ui --strip-components=2 swagger-ui-2.1.4/dist && \
-    rm v2.1.4.tar.gz && \
-    perl -pi -w -e  's{http://petstore.swagger.io/v2/swagger.json}{/api/v1/api-docs}g;' /swagger-ui/index.html
+    tar zxf swagger-ui-v2.1.4.tar.gz -C /swagger-ui --strip-components=2 swagger-ui-2.1.4/dist && \
+    rm swagger-ui-v2.1.4.tar.gz && \
+    sed -i s#http://petstore.swagger.io/v2/swagger.json#/api/v1/api-docs#g /swagger-ui/index.html
```

Comment:
i notice this is also in another pr #3064?

Reply:
#3064 was supposed to only update swagger version